ACCEPTANCE_TESTS.rst

```diff
@@ -0,0 +1,163 @@
+* [ ] Download multipart object as nonseekable binary stream
+
+
+Retry Logic
+~~~~~~~~~~~
+* [x] Retry on connection related errors when downloading data
+* [ ] Compare MD5 to ``ETag`` and retry for mismatches if all following scenarios are met:
+      
+  * If MD5 is available
+  * Response does not have a ``ServerSideEncryption`` header equal to ``aws:kms``
+  * Response does not have ``SSECustomerAlgorithm``
+  * ``ETag`` does not have ``-`` in its value indicating a multipart transfer
+
+
+Extra Parameters
+~~~~~~~~~~~~~~~~
+* [x] Download an object of a specific version
+* [x] Download an object encrypted with sse-c
+* [x] Download an object using requester pays
+
+
+Performance
+~~~~~~~~~~~
+* [ ] Maximum memory usage does not grow linearly with linearly increasing file size for any download.
+* [ ] Maximum memory usage does not grow linearly with linearly increasing number of downloads.
+
+
+Copy Tests
+----------
+
+General
+~~~~~~~
+* [ ] Copy single nonmultipart object
+* [ ] Copy single multipart object
+* [ ] Copy multiple nonmultipart objects
+* [ ] Copy multiple multipart objects
+* [ ] Provide a transfer size for any copy in lieu of using HeadObject.
+* [ ] Failed/cancelled multipart copy is aborted and leaves no orphaned parts
+* [ ] Ctrl-C of any copy does not hang
+
+
+Validation
+~~~~~~~~~~
+* [ ] Before copy, modify chunksize to an acceptable size when needed:
+
+  * [ ] Make chunksize 5 MB when the provided chunksize is less
+  * [ ] Make chunksize 5 GB when the provided chunksize is more
+  * [ ] Increase chunksize till the maximum number of parts for multipart copy is less than or equal to 10,000 parts
+
+* [ ] Before copy, ensure copy is nonmultipart if the file size is less than 5 MB no matter the provided multipart threshold.
+
+
+Extra Parameters
+~~~~~~~~~~~~~~~~
+* [ ] Copy multipart and nonmultipart file with any of the following properties:
+  
+  * [ ] ACL's
+  * [ ] CacheControl
+  * [ ] ContentDisposition
+  * [ ] ContentEncoding
+  * [ ] ContentLanguage
+  * [ ] ContentType
+  * [ ] Expires
+  * [ ] Metadata
+  * [ ] Grants
+  * [ ] StorageClass
+  * [ ] SSE (including KMS)
+  * [ ] Website Redirect
+
+* [ ] Copy multipart and nonmultipart copies with copy source parameters:
+
+  * [ ] CopySourceIfMatch
+  * [ ] CopySourceIfModifiedSince
+  * [ ] CopySourceIfNoneMatch
+  * [ ] CopySourceIfUnmodifiedSince
+
+* [ ] Copy nonmultipart object with metadata directive and do not use metadata directive for multipart object
+* [ ] Copy multipart and nonmultipart objects of a specific version
+* [ ] Copy multipart and nonmultipart objects using requester pays
+* [ ] Copy multipart and nonmultipart objects using a sse-c key
+* [ ] Copy multipart and nonmultipart objects using a copy source sse-c key
+* [ ] Copy multipart and nonmultipart objects using a copy source sse-c key and sse-c key
+
+
+Cross-Bucket
+~~~~~~~~~~~~
+* [ ] Copy single nonmultipart object across sigv4 regions
```

Comment:
We don't do this now. Right? But planning to add it in botocore. So I guess my question is should we be documenting behavior implemented in botocore here in this document?